kratos/geometries/line_2d_3.h

```diff
@@ -73,6 +73,9 @@ class Line2D3
     /// Pointer definition of Line2D3
     KRATOS_CLASS_POINTER_DEFINITION( Line2D3 );
 
+    /// Type of edge geometry
+    typedef Line2D3<TPointType> EdgeType;
```

Comment:
```suggestion
    using EdgeType = Line2D3<TPointType>;
```

I know the rest of this file uses `typedefs` rather than `using` for type aliases, but modernization has to begin *somewhere*.

Reply:
I agree with you that modernization needs to start somewhere. So, in the files that I have touched, I have replaced all occurrences of `typedef` by `using`.